.github/workflows/ruff.yaml

```diff
@@ -0,0 +1,10 @@
+name: Linting
+on: [ push, pull_request ]
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: chartboost/ruff-action@v1
+        with:
+          args: check --fix
```

Comment:
```suggestion
```
Probably don't want to fix on CI, but on the pre-commit hook maybe?

Btw it's a good idea to set your editor (and CI) to ensure files end with a `\n` [[1]](https://stackoverflow.com/a/729795). 

[1] - https://stackoverflow.com/a/729795